docs/modules/gear/actions-triggers.mdx

```diff
@@ -0,0 +1,191 @@
+---
+id: actions-triggers
+title: Actions & Triggers
+---
+
+Actions are a set of features that are applied to players, teams or matches, similiar to [Kits](kits).
+
+##### Action Elements
+
+<div className="table-container">
+  <table>
+    <thead>
+      <tr>
+        <th>Element</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>
+          <label>{`<action> </action>`}</label>
+        </td>
+        <td>A single action.</td>
+      </tr>
+      <tr>
+        <td>
+          <label>{`<trigger/>`}</label>
+        </td>
+        <td>Creates a trigger to cause an Action.</td>
+      </tr>
```

Comment:
Trigger is *not* an action, trigger is what can "trigger an action" based on a dynamic filter changing. This is similar to `give`, `lend` and `take` from kits